<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labelprint_design_save_dialog</class>
 <widget class="QDialog" name="labelprint_design_save_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="outer_stylesheet" stdset="0">
   <string>dialog_hasborder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_path_2">
       <property name="text">
        <string>文件名</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_filename"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button_nowrap" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="icon">
        <iconset resource="peripheral.qrc">
         <normaloff>:/image/ok.png</normaloff>:/image/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button_nowrap" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="icon">
        <iconset resource="peripheral.qrc">
         <normaloff>:/image/exit.png</normaloff>:/image/exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button_nowrap</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button_nowrap.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="peripheral.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>labelprint_design_save_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>labelprint_design_save_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
